src/AvtaleSide/MaalSteg/RedigerMaal/RedigerMaal.tsx

```diff
@@ -108,12 +107,13 @@ class RedigerMaal extends React.Component<Props, State> {
                 </option>
             );
         }
-        const liste = this.props.ledigeMaalkategorier.map((mal, index) => (
-            <option value={mal} key={index}>
-                {mal}
-            </option>
-        ));
-
+        const liste = this.props.ledigeMaalkategorier
+            .filter(mal => mal.toString() !== this.state.valgtKategori)
```

Comment:
Bra med bruk av `filter` her 👍 Men er det nødvendig å gjøre mål om til string (`mal.toString()`)? Sånn jeg kan se det så er både `this.state.valgtKategori` og elementer i lista `this.props.ledigeMaalkategorier` av type `Maalkategori`. Og da kan de vel sammenlignes med `mal !== this.state.valgtKategori`?